src/views/Hero/Hero.scss

```diff
@@ -1,11 +1,57 @@
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
+
 .HeroView {
-  height: 100%;
+  display: flex;
+  overflow: hidden;
+  justify-content: center;
+  min-height: 100%;
```

Comment:
When I inspect the code this style is actually overriden by `min-height: 90%` - do we still need to include this?

Reply:
We probably don't want to remove it here since it's a fallback height. If all Hero components have the same height then we can just set it here once and not override it. 

@ilopezro Thoughts?

Reply:
Correct that's the intended purpose of having the different class names within the hero page as well to prepare for the future views.